test/go/row_batch_test.go

```diff
@@ -108,6 +109,26 @@ func (this *RowBatchTestSuite) TestRowBatchNoPaginationKeyIndex() {
 	this.Require().Equal(false, batch.ValuesContainPaginationKey())
 }
 
+func (this *RowBatchTestSuite) TestBlah() {
+	type Hello struct {
+		Blah string
+		Wah uint64
+	}
+	t := Hello{Blah: "Hello", Wah: uint64(10000000)}
+
+	vals := []ghostferry.RowData{
+		ghostferry.RowData{"hello goodbye"},
+		ghostferry.RowData{100},
+		ghostferry.RowData{t},
+	}
+
+	batch := ghostferry.NewRowBatch(this.sourceTable, vals, 0)
+
+	s := batch.EstimateByteSize()
+
+	fmt.Printf("%d", s)
```

Comment:
Maybe we should actually assert something?

Reply:
Removed since I'm arleady testing in the callback.